src/shell/shell.cpp

```diff
@@ -668,7 +668,20 @@ void SHELL_Init() {
 	        "Type CD drive: to display the current directory in the specified drive.\n"
 	        "Type CD without parameters to display the current drive and directory.\n");
 	MSG_Add("SHELL_CMD_CLS_HELP","Clear screen.\n");
-	MSG_Add("SHELL_CMD_DIR_HELP","Directory View.\n");
+	MSG_Add("SHELL_CMD_DIR_HELP","Displays a list of files and subdirectories in a directory.\n");
+	MSG_Add("SHELL_CMD_DIR_HELP_LONG","DIR [drive:][path][filename] [/[W|B]] [/P] [/[AD]|[A-D]] [/O[-][N|E|S|D]]\n\n"
+		   "  [drive:][path][filename]\n"
+		   "              Specifies drive, directory, and/or files to list.\n"
+		   "  /W          Uses wide list format.\n"
+		   "  /B          Uses bare format (no heading information or summary).\n"
+		   "  /P          Pauses after each screenful of information.\n"
+		   "  /AD         Displays all directories.\n"
+		   "  /A-D        Displays all files.\n"
+		   "  /O          List by files in sorted order.\n"
+		   "               -  Prefix to reverse order\n"
+		   "  sortorder    N  By name (alphabetic)       S  By size (smallest first)\n"
+		   "               E  By extension (alphabetic)  D  By date & time (earlist first)\n"
```

Comment:
Here's some out-of-context snippets grepped from `ls`:

``` text
                               otherwise: sort by ctime, newest first

  -S                         sort by file size, largest first

      --sort=WORD            sort by WORD instead of name: none (-U), size (-S),

  -t                         sort by modification time, newest first

  -u                         with -lt: sort by, and show, access time;
                               with -l: show access time and sort by name;
                               otherwise: sort by access time, newest first
```

Reply:
> List by files in sorted order sounds weird

May be weird but that is what `dir` outputs in cmd on Windows 10. I don't have real DOS installed to know if that differs. Which is better, use the original text from DOS or enhance it for modern readability?

Reply:
I used the exact same wording for this one as in DOS/Windows. Perhaps just keep it as is?

Reply:
OK :) Maybe it's just me (but I am not a native English speaker); if anyone else will find it weird or confusing, then we can always adjust the wording later.